pkg/cli/cmd/bicep/publish/publish.go

```diff
@@ -167,14 +171,40 @@ func (r *Runner) Run(ctx context.Context) error {
 	r.Destination = dest
 
 	err = r.publish(ctx)
-	if err != nil {
-		return clierrors.MessageWithCause(err, "Failed to publish Bicep file %q to %q.", r.File, r.Target)
+	var httpErr *errcode.ErrorResponse
+	if errors.As(err, &httpErr) {
+		message := fmt.Sprintf("Failed to publish Bicep file %s to %s", r.File, r.Target)
```

Comment:
```suggestion
		message := fmt.Sprintf("Failed to publish Bicep file %q to %q", r.File, r.Target)
```

This should use quotes to surround the user-specified details like the filename and destination.